core/src/main/java/org/jdbi/v3/core/internal/JdbiClassUtils.java

```diff
@@ -22,8 +28,32 @@ public static boolean isPresent(String klass) {
         try {
             Class.forName(klass);
             return true;
-        } catch (ReflectiveOperationException ignored) {
+        } catch (ReflectiveOperationException | SecurityException ignored) {
```

Comment:
How can you get a `SecurityException` here? From my reading of the `Class.forName` javadoc, this does not seem possible.